vnext/Microsoft.ReactNative/Views/Image/ReactImageBrush.cpp

```diff
@@ -100,7 +101,9 @@ void ReactImageBrush::UpdateCompositionBrush(bool forceEffectBrush) {
       // "You can compute the blur radius of the kernel by multiplying the standard deviation by 3.
       // The units of both the standard deviation and blur radius are DIPs.
       // A value of zero DIPs disables this effect entirely."
-      compositionBrush.Properties().InsertScalar(BlurBlurAmount, m_blurRadius / 3);
+      if (m_blurRadius > 0) {
```

Comment:
I dont know this code.  But can blurRadius be non-zero, then on a subsequent render be zero?  -- Would this value need to be removed in that case?

Reply:
Actually looks like this function is creating the compBrush.. so probably not an issue here.

Reply:
If it changed from non-zero to zero then we'll force the creation of a new effect brush, so the properties will be reset.

Reply:
https://github.com/microsoft/react-native-windows/pull/7222/files#diff-85a95b36797ca57b21e2e39c5e678c0ab8a0026d716b1dce56de2311e11f3143R49